test/jvm/src/test/scala/zio/test/TimeVariantsSpec.scala

```diff
@@ -0,0 +1,30 @@
+package zio.test
```

Comment:
I am pretty sure I am not suppose to put this test here.
Can you tell me where to put it?

Reply:
I think you are looking for `GenSpec.scala`.